package/bpfcountd/Makefile

```diff
@@ -0,0 +1,34 @@
+include $(TOPDIR)/rules.mk
```

Comment:
Move this package to OpenWrt packages (I See that LICENCE identifies is missing).